main/src/ca/uwaterloo/flix/api/lsp/provider/CompleteProvider.scala

```diff
@@ -38,10 +38,52 @@ object CompleteProvider {
+    CompletionItem("override", "override", None, Some("keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, Nil),
+    CompletionItem("pub", "pub", None, Some("keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, Nil),
+    CompletionItem("Pure", "Pure", None, Some("keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, List("=")),
+    CompletionItem("Record", "Record", None, Some("keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, Nil),
+    CompletionItem("ref", "ref", None, Some("keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, Nil),
+    CompletionItem("rel", "rel", None, Some("keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, Nil),
+    CompletionItem("Schema", "Schema", None, Some("keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, Nil),
+    CompletionItem("sealed", "sealed", None, Some("keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, Nil),
+    CompletionItem("select", "select ", None, Some("keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, List("{")),
+    CompletionItem("solve", "solve", None, Some("keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, Nil),
+    CompletionItem("spawn", "spawn", None, Some("keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, Nil),
+    CompletionItem("true", "true", None, Some("keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, List(";")),
+    CompletionItem("type", "type", None, Some("keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, Nil),
+    CompletionItem("unlawful", "unlawful", None, Some("keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, Nil),
+    CompletionItem("with", "with", None, Some("keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, Nil),
 
-    // Keyword-like (e.g. println):
-    CompletionItem("println", "println", None, None, CompletionItemKind.Keyword, InsertTextFormat.PlainText, List("(", ")")),
+    CompletionItem("Bool", "Bool", None, Some("data type"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, Nil),
+    CompletionItem("Char", "Char", None, Some("data type"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, Nil),
+    CompletionItem("Int8", "Int8", None, Some("data type"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, Nil),
+    CompletionItem("Int32", "Int32", None, Some("data type"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, Nil),
+    CompletionItem("Int64", "Int64", None, Some("data type"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, Nil),
+    CompletionItem("String", "String", None, Some("data type"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, Nil),
+    CompletionItem("Unit", "Unit", None, Some("data type"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, Nil)
```

Comment:
Instead of "data type" I think we should just say "type" (similar to how we just say "keyword")